Makefile

```diff
@@ -52,8 +53,22 @@ ARTIFACTS ?= $(PROJECT_DIR)/bin
 # Refer to https://github.com/GoogleContainerTools/distroless for more details
 BASE_IMAGE ?= gcr.io/distroless/static:nonroot
 BUILDER_IMAGE ?= golang:$(GO_VERSION)
+# These are used for OpenShift builds.
+OCP_BASE_IMAGE ?= registry.access.redhat.com/ubi9/ubi-micro:latest
+OCP_BUILDER_IMAGE ?= brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.23
 CGO_ENABLED ?= 1
 
+# Conditional to decide which build to use.
+ifeq ($(USE_OCP),true)
```

Comment:
I would prefer default builds on OCP.

Reply:
I added this to avoid creating multiple targets.

Reply:
I just mean that our default setting without this flag should build OCP.

This may make konflux integration a bit easier. Otherwise you have to set these environment variables before building the konflux image.

Reply:
I don't think we gonna use the Make commands in konflux, so we are good there.